package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@duckduckgo/autoconsent": "^5.1.0",
-    "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#8.4.2",
+    "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#abrown/android-incontext-email-promotion",
```

Comment:
This will be changed to a stable release version before merging PR (waiting on it being made currently)